src/main/scala/org/clulab/timenorm/scate/TemporalNeuralParser.scala

```diff
@@ -163,7 +163,7 @@ class TemporalNeuralParser(modelStream: Option[InputStream] = None) extends Auto
   }
 
   def parseBatchToXML(text: String, spans: Array[(Int, Int)]): Array[Elem] = {
-    val antixmlCleanedText = """[^\u0009\u000A\u000D\u0020-\uD7FF\uE000-\uFFFD]""".r.replaceAllIn(text, " ")
+    val antixmlCleanedText = """[^\u0009\u000A\u000D\u0020-\uD7FF\uE000-\uFFFD]""".r.replaceAllIn(text, " " * _.group(0).length)
```

Comment:
There needs to be a test for the case that triggered this, i.e., make sure the test fails without this change and succeeds with it.